### ISSUE TITLE
Nether, End and Resource resets...

### ISSUE THREAD

**Issue by GoddessTT** [ResqDiver1317/AdultCraft-Foundations#36]
I did the End reset on 11/3, I was going to do nether (resource was on hold) but you decided to perhaps put it on a schedule for automation.

**Comment by GoddessTT** [comment]
I know you got busy, did you want me to do the nether? Meesha has an open ticket on it.

**Comment by ResqDiver1317** [comment]
I don't think we need to do them monthly, I think every 2 months would prob be good (except maybe the end). I don't want to start it right now as I have a plan I need to pull the pre-move sauce files from that server before we try to work on worlds there. I'm also looking into a feature of the pregen mod that may retrogen/restore the worlds without completely resetting them. I'm going to file a ticket with them shortly about it to make sure I understand that feature. If thats the case, then this process just got WAY easier as I can automate that in the Multicraft panel. If it won't work like I'm hoping, then I will post instructions on how to do resets and we will decide set days to do them on.
